web-ui/src/main/resources/catalog/components/search/searchmanager/SearchFormDirective.js

```diff
@@ -123,9 +123,9 @@
 
       $scope.searching++;
       $scope.searchObj.params = angular.extend({},
-          $scope.searchObj.defaultParams || defaultParams,
```

Comment:
In ascending priority order:
Before:
* Inherited default parameters
* Current search parameters.
* Directive default parameters

Now:
* Inherited default parameters
* Directive default parameters
* Current search parameters.